src/addrman.h

```diff
@@ -504,14 +504,14 @@ friend class CAddrManTest;
         // so we store all bucket-entry_index pairs to iterate through later.
         std::vector<std::pair<int, int>> bucket_entries;
 
-        for (int bucket = 0; bucket < nUBuckets; bucket++) {
-            int nSize = 0;
-            s >> nSize;
-            for (int n = 0; n < nSize; n++) {
-                int nIndex = 0;
-                s >> nIndex;
-                if (nIndex >= 0 && nIndex < nNew) {
-                    bucket_entries.emplace_back(bucket, nIndex);
+        for (int bucket = 0; bucket < nUBuckets; ++bucket) {
+            int bucket_size{0};
+            s >> bucket_size;
```

Comment:
Just to clarify... this is the number of entries in this bucket, right? Not the capacity of the bucket, which is `ADDRMAN_BUCKET_SIZE`?

Reply:
Exactly (see the comment above about serialization format: https://github.com/jnewbery/bitcoin/blob/dbdf63c6d82ae8d1ca73f11d1b8844614f0bf8e3/src/addrman.h#L341)

Reply:
I just thought both being called "size" is a bit confusing so `num_entries` would have been my choice for naming, but it's the nittiest of nits, non-blocking

Reply:
I agree that this is confusing. I've renamed to `num_entries`.